### ISSUE TITLE
The white paper too technical 

### ISSUE THREAD

**Issue by Mataashi** [calistus-igwilo/nitda-blockchain-scholarship#33]
The chapter1 was very educating and eye opening MashaAllah although the white paper is a bit too technical to comprehend, i assume with more knowledge and examples of the cryptographic vocabulary it can also be as interesting as the chapter1.

**Comment by harzjunior** [comment]
> The chapter1 was very educating and eye opening MashaAllah although the white paper is a bit too technical to comprehend, i assume with more knowledge and examples of the cryptographic vocabulary it can also be as interesting as the chapter1.

Yes! the technology is simply amazing and yet so fascinating. Just like you said With time, you will get to know more about the technology. The key to learning a new tech is by reading and reading until you get hang of it :)
